src/components/mongo.ts

```diff
@@ -75,6 +78,14 @@ export class Mongo extends pulumi.ComponentResource {
     this.registerOutputs();
   }
 
+  private createRandomPassword() {
+    return new random.RandomPassword(`${this.name}-db-password`, {
+      length: 16,
+      overrideSpecial: '_%$',
+      special: true,
+    }).result;
```

Comment:
```suggestion
    const password = new random.RandomPassword(`${this.name}-db-password`, {
      length: 16,
      overrideSpecial: '_%$',
      special: true,
    });
    return password.result;
```
Feel free to ignore this readability improvement if you don't like it.